tools/incident.go

```diff
@@ -12,7 +12,7 @@ import (
 type ListIncidentsParams struct {
 	Limit  int    `json:"limit" jsonschema:"description=The maximum number of incidents to return"`
 	Drill  bool   `json:"drill" jsonschema:"description=Whether to include drill incidents"`
-	Status string `json:"status" jsonschema:"description=The status of the incidents to include"`
+	Status string `json:"status" jsonschema:"description=The status of the incidents to include. Valid values: 'active', 'resolved', 'mitigated'"`
```

Comment:
I think `active` and `resolved` are the only standard ones. I think I am ok merging this, but I wonder if we could populate this on startup in the future by fetching the customized values.